src/sentry/lang/javascript/plugin.py

```diff
@@ -44,9 +44,7 @@ def get_event_preprocessors(self, data, **kwargs):
         return []
 
     def get_stacktrace_processors(self, data, stacktrace_infos, platforms, **kwargs):
-        if data.get("processed_by_symbolicator", False) and not data.get(
-            "symbolicator_stacktraces"
-        ):
+        if data.get("processed_by_symbolicator", False):
```

Comment:
Do we even still need `processed_by_symbolicator` at this point?

Reply:
I'd drop it. We already have all other meta fields that only symbolicator produces

Reply:
I  will double-check, as we already had bugs related to putting a mixed-stacktrace event through the two different symbolicators.

Reply:
Yes, I believe we should keep the current logic as is, and revisit this in https://github.com/getsentry/sentry/pull/49675.
That PR already has merge conflicts, and will probably also conflict with this one here.

But once we have that done, we should be able to simplify this logic a bit.